content/docs/support-and-ops/ops-recipes/balance-workloads.md

```diff
@@ -0,0 +1,176 @@
+
+  ```yaml
+  strategies:
+    "LowNodeUtilization":
+      enabled: true
+      params:
+        nodeResourceUtilizationThresholds:
+          thresholds:
+            cpu: 50
+            memory: 50
+            pods: 10
+          targetThresholds:
+            cpu: 75
+            memory: 75
+            pods: 50
+  ```
+
+  - **Thresholds**: Define the minimum utilization level below which a node is considered underutilized.
+  - **TargetThresholds**: Define the desired utilization level after redistribution.
+
+### 4. **HighNodeUtilization**
+
+- **Purpose**: Evicts pods from overutilized nodes to reduce load and prevent resource saturation.
+- **Configuration**:
+
+  ```yaml
+  strategies:
+    "HighNodeUtilization":
+      enabled: true
+      params:
+        nodeResourceUtilizationThresholds:
+          thresholds:
+            cpu: 80
+            memory: 80
+            pods: 80
+  ```
+
+  - **Thresholds**: Define the maximum utilization level above which a node is considered overutilized.
+
+### 5. **RemovePodsViolatingNodeAffinity**
+
+- **Purpose**: Evicts pods that are scheduled on nodes in violation of their node affinity rules.
+- **Configuration**:
+
+  ```yaml
+  strategies:
+    "RemovePodsViolatingNodeAffinity":
+      enabled: true
+  ```
+
+### 6. **RemovePodsViolatingInterPodAntiAffinity**
+
+- **Purpose**: Evicts pods violating inter-pod anti-affinity rules to ensure proper distribution according to these rules.
+- **Configuration**:
+
+  ```yaml
+  strategies:
+    "RemovePodsViolatingInterPodAntiAffinity":
+      enabled: true
+  ```
+
+### 7. **RemovePodsViolatingTopologySpreadConstraint**
+
+- **Purpose**: Evicts pods that violate topology spread constraints, which ensure even distribution across specified topology domains.
+- **Configuration**:
+
+  ```yaml
+  strategies:
+    "RemovePodsViolatingTopologySpreadConstraint":
+      enabled: true
+  ```
+
+2. Deploy the app from the [Giant Swarm app catalog](https://github.com/giantswarm/descheduler-app):
+
+```sh
+export ORG=<your-org>
+export CLUSTER=<your-cluster>
+kubectl gs template app \
+  --catalog giantswarm \
+  --name hello-world-app \
+  --target-namespace org-$ORG \
+  --organization $ORG \
+  --cluster-name $CLUSTER \
+  --version 1.0.0 > descheduler-app.yaml
+```
+
+In case of customize the profile, save your config values to a file:
+
+```sh
+cat <<EOF > /tmp/values.yaml
+deschedulerPolicy:
+  profiles:
+    - name: default
+      pluginConfig:
+        - name: "DefaultEvictor"
+          args:
+            ignorePvcPods: true
+            ...
+EOF
+```
+
+Then add the flag `--user-config=/tmp/values.yaml` to the previous command.
+
+And apply it to the platform API (aka MC API):
+
+```sh
+kubectl apply -f descheduler-app.yaml
+```
+
+Now the scheduler cronjob has been created and soon it will spawn a new job. By default, the cronjob runs every 2 minutes. **Since we want to using once, we will remove the app after the job is done.**
```

Comment:
```suggestion
Now the scheduler cronjob has been created and soon it will spawn a new job. By default, the cronjob runs every 2 minutes. **Since we want to use it once, we will remove the app after the job is done.**
```